Autopilot/TelemetryManager/Src/telemetryStateClasses.cpp

```diff
@@ -39,6 +52,9 @@ telemetryState& obtainDataMode::getInstance()
 void decodeDataMode::execute(telemetryManager* telemetryMgr)
 {
     //decode data with Mavlink
+    //Call Jingting's function here
+
+    //State change:
```

Comment:
Another thing to add here: formatting the data so it fits in the struct (unless this is already done automatically)

Reply:
Jinging's function should be sending the data out in the same struct, so I think we're good for that.